web/src/js/ducks/flows.js

```diff
@@ -20,6 +21,7 @@ const defaultState = {
     view: [],
     filter: undefined,
     highlight: undefined,
+    sort: {sortColumn: undefined, sortDesc: false},
 }
 
```

Comment:
1) Add `makeSortFn` here. Without testing, I hope we can implement it by exporting all Columns in `flowtable-columns.js` and then just using this:

```javascript
import Columns from "../components/flowtable-columns"

function makeSortFn(sort){
    return Columns[sort.sortColumn].sortKeyFun
}
```

(all_columns in that file is also an relict of the past where we didn't have ES6 modules and could be replaced similarly)